server/controllers/threads.controller.js

```diff
@@ -203,14 +231,28 @@ function show(req, res, next) {
                 },
             });
         });
-        thread.getMessages({
+
+        // Filter by date if start_date query param is given
+        const where = {};
+        if (req.query.start_date) {
+            const startDate = new Date(req.query.start_date);
+            where.where = {
+                updatedAt: {
```

Comment:
I would change this from `updatedAt` to `createdAt`. Few explainations, we never want to change ordering of messages regardless if we add functionality to edit messages (think how a forum allows edits) and we want the chronological order to remain the same, if you think of a messaging service like slack or facebook messenger.

Reply:
Sure that makes sense, I'll make this change now.